docker/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM australia-southeast1-docker.pkg.dev/analysis-runner/images/driver:62e79e54d43aa1762ec68a6661872320a39517bd-hail-2ea2615a797a5aff72d20d9d12a2609342846a07
+
+ARG VERSION=4.8.13
+
+RUN python3 -m pip install cpg_utils[workflows]==$VERSION
```

Comment:
Won't this result in a race condition with the package generation + publishing?

Should we instead build the package locally within the Docker container by copying the files from the checked out repo?

Reply:
Good point! I'll change that

Reply:
Done